### PR TITLE
feat: use assembly script lib in noedjs

### PR DESCRIPTION
**Pull request by ajwootto**
- use assembly script lib in nodejs sdk
- import it dynamically during initialization of the client
- update variable and feature calls to work if the client hasn't finished initializing

### CODE REVIEW

File: lib/shared/bucketing-assembly-script/assembly/types/platformData.ts [hunk 1]
```diff
@@ -4,7 +4,6 @@ import { getStringFromJSON } from '../helpers/jsonHelpers'
 export class PlatformData extends JSON.Obj {
     platform: string
     platformVersion: string
-    deviceModel: string
```
Comment (jonathannorris) [inline lib/shared/bucketing-assembly-script/assembly/types/platformData.ts:7]: Why did you remove it from here?

Comment (ajwootto) [reply lib/shared/bucketing-assembly-script/assembly/types/platformData.ts:7]: because deviceModel isn't platform data, it's user data. In a backend context the "device" is associated with the current user being processed, it's not a platform-level thing

**Review by jonathannorris — APPROVED** [review]
